src/lib.rs

```diff
@@ -345,6 +357,18 @@ fn show_eshop(_lua_state: *const u8) {
     menus::show_main_menu();
 }
 
+#[skyline::hook(offset = 0x3778bf4, inline)]
+unsafe fn text_inline(ctx: &mut InlineCtx) {
+    let msbt_label = skyline::from_c_str((ctx as *const InlineCtx as *const u8).add(0x100).add(224));
+    let mut text: Vec<u16> = "jozz?\0".to_string().encode_utf16().collect();
```

Comment:
There is no need to perform this every time a label is provided, only when it's the one we need. So you can move it in the check down below

Reply:
Actually this goes unused lol, remove it